package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "loopback.io-workflow-scripts",
   "version": "1.0.0",
-  "description": "This file: Node-dependant workflow scripts for the jekyll-based site",
+  "description":
+    "This file: Node-dependant workflow scripts for the jekyll-based site",
   "scripts": {
     "start": "bundle exec jekyll serve --incremental",
-    "fetch-readmes": "get-readmes --repos=./_data --out=./_includes/readmes",
+    "fetch-readmes":
+      "./update-readmes.sh --repos=./_data --out=./_includes/readmes && ./update-v4-readmes.sh",
```

Comment:
Please keep key/value pairs on a single line, that's what `npm` will do the next time we run any `npm` command modifying `package.json`. The same comment applies to `description` too.

Reply:
My apologies. That was auto-formatted on save by editor (Prettier extension) as I didn't have config to disable it from `.json` files. I didn't realize this had happened. 

I was going to make a PR to fix it but turns out the Travis build already did so in the last build https://github.com/strongloop/loopback.io/commit/5871ba370f189c3bd15e276396a1146da8c580b6